src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/takecard/TakeCard.kt

```diff
@@ -8,78 +8,55 @@ import javafx.beans.property.SimpleBooleanProperty
 import javafx.event.ActionEvent
 import javafx.geometry.Pos
 import javafx.scene.control.Button
+import javafx.scene.control.Label
 import javafx.scene.layout.AnchorPane
 import javafx.scene.layout.Pane
 import javafx.scene.layout.Priority
 import javafx.scene.paint.Color
 import org.wycliffeassociates.otter.common.data.model.Take
 import org.wycliffeassociates.otter.common.device.IAudioPlayer
+import org.wycliffeassociates.otter.jvm.app.widgets.SimpleAudioPlayer
+import org.wycliffeassociates.otter.jvm.app.widgets.simpleaudioplayer
 import tornadofx.*
 
 class TakeCard(val take: Take, player: IAudioPlayer) : AnchorPane() {
     val playedProperty = SimpleBooleanProperty(take.played)
-    private val badge = stackpane {
-        // custom css class
-        style {
-            backgroundRadius += box(0.px, 10.px, 0.px, 10.px)
-            padding = box(8.px)
-        }
-        addClass("badge")
+    val badge = stackpane {
+        addClass(TakeCardStyles.badge)
         val icon = MaterialDesignIconView(MaterialDesignIcon.CREATION, "18px")
```

Comment:
here too, with the icon?  I think we could move this in with the TakeCardStyles alongside the `play()` and `pause()` icons

Reply:
Yep good call